src/backend/bottles/utils.ts

```diff
@@ -0,0 +1,114 @@
+import { spawn } from 'child_process'
+import { logDebug, LogPrefix } from '../logger/logger'
+import { isFlatpak } from '../constants'
+import shlex from 'shlex'
+
+function prepareBottlesCommand(
+  args: string[],
+  bottlesBin: string,
+  noCli?: boolean,
+  join?: boolean
+): string[] | string {
```

Comment:
I wouldn't recommend adding options like this that alter the return type of a function. Type checking gets annoying with that rather fast, since TS can't know when you're returning a `string` and when it's a `string[]`
Just a suggestion of course, but maybe do the `shlex.join` in the calling function (or in a small wrapper) & always return a `string[]` here